ui/src/components/Configuration/ReqDefinitionForm.vue

```diff
@@ -150,6 +158,8 @@ const reqDefinition = reactive(State)
 
 const minVal = ref(1)
 const count = ref(0)
+const actionsAddIcon = ref(markRaw(actionsAdd));
```

Comment:
Can't you just use the Icons directly? 

I tried this branch locally to take out these two lines with markRaw/ref and just use the icons directly from featherds and it seemed to work okay, so I'm curious as to why these two steps were added, maybe there's something I'm missing.

Reply:
Done, Changed to simple icon only.